recipe/meta.yaml

```diff
@@ -34,8 +34,11 @@ requirements:
     - arrow-cpp {{ version }}
 
   run:
+    - glog
+    - snappy
     - boost-cpp
     - python
+    - setuptools
     - {{ pin_compatible('numpy', lower_bound='1.14') }}
     - pandas
```

Comment:
Pandas is not a hard dependency anymore, but I suggest to remove it in the next release.